src/index.js

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+const state = {
+  city: 'Portland, OR',
+  temperature: 72,
+  latitude: 45.5152,
+  longitude: 122.6784,
+};
+
+let getCityName;
+let curWeatherEmojis = document.getElementById("curWeatherEmojis");
+let cityDisplayOnLoad = document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${state.city}`;
+
+const updateCityWhileTyping = () => {
+  const input = document.getElementById("defaultCity").value;
+  document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${input}`
+  state.city = input;
+};
+
+document.getElementById("sky").addEventListener('change', (event) => {
+  const bgValue = event.target.options[event.target.selectedIndex].value;
+  if( bgValue == 'sunny' ) {
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";      
+    } else if( bgValue == "cloudy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";      
+    } else if( bgValue == "rainy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";      
+    } else if( bgValue == "snowy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";      
+    }
+});
+
+const weatherEmojisandColor = () => {
```

Comment:
Consider renaming this method to use a verb to indicate that this method does something, maybe something like `setWeatherEmojisaAndColor`